services/ui-src/src/components/statusing/TableRow.test.tsx

```diff
@@ -37,24 +37,18 @@ const tableRowComponent = ({ page, rowDepth }: RowProps) => {
   );
 };
 
-describe("Test TableRow", () => {
-  beforeEach(() => {
-    mockedUseStore.mockReturnValue(mockUseStore);
-    render(tableRowComponent({ page: mockTableRowPage, rowDepth: 1 }));
-  });
+describe("<TableRow />", () => {
+  describe("Test TableRow", () => {
```

Comment:
do we need the nested describe if there's only one block?

Reply:
The second `describe` block is there because there's a `beforeEach` block used with that test and `testA11y` needs to be isolated from it. But, in this case what's in the `beforeEach` block can be moved inside the test since there's only one.